ansible/roles/local_files/tasks/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+- name: extract remote archive to local_files directory
+  unarchive: copy=no src="{{ local_files_archive }}" dest="local_files/"
```

Comment:
There is actually a site-wide variable, `local_files_dir` (defined in `ansible/site_vars.yml`) that designates the folder used for storing local files.  You should thus use `dest="{{ local_files_dir }}"` instead of hard-coding `dest="local_files/"`.